src/NovelImageRect.cpp

```diff
@@ -27,25 +44,43 @@ NovelImageRect::NovelImageRect(NovelLayeringService* layeringService,
                                const GeoVector<float>& size,
                                const NovelCommonArgs& args,
                                GLuint programId,
-                               const RGBAConfig& colourTint) :  NovelRenderObject(layeringService, screenScale, size, args, programId),
-_imageDir(""), _colourTint(colourTint) {
+                               const RGBAConfig& colourTint) : NovelRenderObject(layeringService,
```

Comment:
Why repeat all this logic, rather than just forwarding to the above constructor with an empty `imageDir`?

Reply:
Fixed.